lib/bolt/outputter.rb

```diff
@@ -14,6 +14,15 @@ def self.for_format(format)
     def initialize(stream = $stdout)
       @stream = stream
     end
+
+    def replace_undef(params)
+      params.map do |_, v|
+        t = v['type']
+        if defined?(t.name) && t.name == 'Data'
```

Comment:
`defined?(t.name)` should be `t.respond_to?(:name)`

What case will it not respond to `name`?

Reply:
I think its because the spec tests insert a String.  I'll hunt those down and put a double in place.